### ISSUE TITLE
wrong distance between locations

### ISSUE THREAD

**Issue by roebert** [wildlyinaccurate/trip-planner#7]
when i try it on your site: https://wildlyinaccurate.com/trip-planner/ the calculated distances between even change! i tried a route from: chemnitz to nürnberg to hamburg to basel to chemnitz
take a look at the distances when adding the waypoints - they change! and some distances cant be right

**Comment by wildlyinaccurate** [comment]
Yes, I've seen this before... I never figured out why it happens, but it is really annoying especially if you have spent some time entering all the locations.

I'm hoping to re-write this app soon because I think the current implementation has some strange behaviour. But for now, I don't have a fix for this bug -- sorry :disappointed:
